### PR TITLE
Fix / Update for Use Billing name on 'Guest' checkout

### PR DESCRIPTION
**Pull request by haroon-chaudhry**

### CODE REVIEW

File: Helper/ApiClient.php [hunk 1]
```diff
@@ -92,7 +92,12 @@ public function oauthAuthentication($storeId)
   public function prepareOrderData($order) 
   {
     $data['email'] = $order->getCustomerEmail();
-    $data['customer_name'] = $order->getCustomerName();
+	$customer = $order->getCustomerName();
```
Comment (omryYotpo) [inline Helper/ApiClient.php]: please change naming to $customer_name

File: Helper/ApiClient.php [hunk 2]
```diff
@@ -92,7 +92,12 @@ public function oauthAuthentication($storeId)
...
+	if($customer == "Guest"){
+		$customer = $order->getBillingAddress();
```
Comment (omryYotpo) [inline Helper/ApiClient.php]: please use a different variable to save the result and name it something like billing_address

Comment (FarhanIslamIDT) [inline Helper/ApiClient.php]: Done in d309393

File: Helper/ApiClient.php (hunk 1 above)
Comment (FarhanIslamIDT) [inline Helper/ApiClient.php]: Done in d309393

**Pull request closed without merging**